src/api/app/assets/javascripts/webui2/patchinfo.js

```diff
@@ -0,0 +1,118 @@
+function setupPatchinfo() { // jshint ignore:line
```

Comment:
[nitpick] We should consider alternatives which doesn't involved that much JavaScript

Reply:
I managed to reduce the amount of JS with rails-ujs, but it's true that this form depends a lot on JS. What could we do instead? For the AJAX/click/change events, I don't really see a way to avoid that. Regarding the `addIssues` function and appending HTML, this could be improved with templates like Mustache, but it's not something we do currently.